src/scss/players.scss

```diff
@@ -360,7 +360,8 @@ $eloValues: (1500 0.3 1 0.3) (1600 0.3 1 0.5) (1849 0 1 0.7) (1850 1 1 0.7) (190
 		.card-container.flipped {
 			transform: rotateY(180deg);
 		}
-		.player-number.leftgame {
+		.player-number.leftgame,
+		.player-number.isDead {
```

Comment:
I like the design pattern of different classes for different uses but shouldn't there be a visual distinction of dead and still sitting? Or is this to prevent the metagaming of knowing when someone leaves after being shot?

Reply:
Yes, the intent was to prevent meta gaming when someone gets shot and leaves, icon will still show for people not in the lobby. I figured it would be better for people not in a lobby to still see that this player is still in a lobby.

Also, it's been a while so I do not remember if I tested that or not, but it's probably a good idea to test if it still works properly when the player quits the entire site.